index.html

```diff
@@ -0,0 +1,127 @@
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Log out</a></li>
+            </ul>
+        </nav>
+        <nav class="bar2">
+            <div class="menu" id="menu"></div>
+            <a href="index.html"><img class="logo" src="img/iot.gif" alt="logo"></a>
+                    <ul class="sec-ul"  id="mobile-menu">
+                        <li><a class="close-btn" id="close-btn">X</a></li>
+                        <li><a href="aboutpage.html" class="mobilemenu">About</a></li>
+                        <li><a href="index.html" class="mobilemenu">Program</a></li>
+                        <li><a href="#" class="mobilemenu">Join</a></li>
+                        <li><a href="#" class="mobilemenu">Sponsor</a></li>
+                        <li><a href="#" class="mobilemenu">News</a></li>
+                        <li><a href="#" class="button mobilemenu">Iot Campaign</a></li>
+                    </ul>
+        </nav>
+    </header>
+    <section class="section1">
+        <h2 class="h21">
+           Hey there! Welcome
+        </h2>
+        <h2 class="h22">Internet of things Network Conference 2022</h2>
+        <p class="p1">We are Iot network hub. We contribute to the world by created renowed Iot prjoject which saves our planet. 
+            Our mission is to trained spread the good news about how Iot is redefining 
+            the future of tech</p>
+            <p class="bold">
+                <strong>2022.10.15(THU)~ 16(FRI)
+          </strong> </p>
+                 <p class="bottom-text">@ Accra International Conference Center</p>
+    </section>
+    <section class="section2">
+        <h2 class="h23">
+            Main Program
+        </h2>
+         <div class="underline"></div>
+
+            <div class="main-program"></div>
+                <ul class="blocks">
+                    <li class="b"><i class="C"><img src="img/1.png" alt=""></i>
+                        <p class="A">Seminars</p>
+                        <p class="B">we offer seminars to techs every two months
+                            per year. We attach importance to collabos
+                        </p>
+                    </li>
+                    <li class="b"><i class="C"><img src="img/2.png" alt=""></i>
+                        <p class="A">Lectures</p>
+                        <p class="B">Learn mordern techniques
+                            in our daily classes from monday to friday
+                            each week
+                        </p>
+                    </li>
+                    <li class="b"><i class="C"><img src="img/3.png" alt=""></i>
+                        <p class="A">Programs</p>
+                        <p class="B">Enrol and become eligible for our
+                            government sponsored susidies
+                        </p>
+                    </li>
+                    <li class="b"><i class="C"><img src="img/4.png" alt=""></i>
```

Comment:
- [ ] The padding around the program section does not match the original design. Kindly add padding to make it match the original design
- [ ] The cards are also not the same size as the original design. Please ensure that they follow the design guidelines.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/169508134-4a9166fe-bcaf-498a-8efd-3708e5ce1cd8.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/169508210-1406df02-9ca8-4085-84f5-677d20499376.png)